validations/transfer.test.ts

```diff
@@ -85,7 +86,19 @@ describe('/transfer', () => {
           transferInParams,
         )
         expect(duplicateTransferResponse.isOk).to.be.true
-        expect(duplicateTransferResponse.unwrap().transferId).to.equal(transferInResponse.unwrap().transferId)
+        expect(duplicateTransferResponse.unwrap().transferId).to.equal(
+          transferInResponse.unwrap().transferId,
+        )
+
+        const transferStatusResponse =
```

Comment:
similarly, probably better to have this check before the duplicateTransfer check